chart/templates/servicemonitor.yaml

```diff
@@ -15,10 +15,15 @@ metadata:
   {{- end }}
 spec:
   selector:
-    app.kubernetes.io/name: {{ include "app.name" . }}
-    app.kubernetes.io/instance: {{ .Release.Name }}
-    helm.sh/from: deploy.{{ include "app.fullname" . }}
+    matchLabels:
+      app.kubernetes.io/name: {{ include "app.name" . }}
+      app.kubernetes.io/version: {{ .Chart.AppVersion }}
+      helm.sh/chart: {{ include "app.chart" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+      app.kubernetes.io/managed-by: {{ .Release.Service }}
+      helm.sh/from: deploy.{{ include "app.fullname" . }}
+      {{- toYaml .Values.service.labels | indent 6 }}
```

Comment:
I think you need `nindent` in here as well

Reply:
fixed